src/components/MultiEmailInput/constants.jsx

```diff
@@ -68,12 +81,30 @@ const CustomValueContainer = ({ children, ...props }) => {
   );
 };
 
-const CustomClearIndicator = prop => (
-  <components.ClearIndicator {...prop}>
+const CustomClearIndicator = props => (
+  <components.ClearIndicator
+    {...props}
+    innerProps={{ ...props.innerProps, ["data-cy"]: "clear-all-button" }}
+  >
     <Close className="cursor-pointer" size={16} />
   </components.ClearIndicator>
 );
 
+const SelectContainer = props => (
+  <components.SelectContainer
+    {...props}
+    innerProps={{
+      ...props.innerProps,
+      ["data-cy"]: "multi-email-select-container",
+    }}
+  >
+    {props.children}
+  </components.SelectContainer>
```

Comment:
Since we are spreading the props, we don't have to explicitly pass the children.

```suggestion
  <components.SelectContainer
    {...props}
    innerProps={{
      ...props.innerProps,
      ["data-cy"]: "multi-email-select-container",
    }}
  />
```